### DISCUSSION TITLE
RIP?

### DISCUSSION THREAD

**Discussion by Cooksey99** [humanetes-com/proof-of-positive-interaction#43] [General]
I know @virgiliolino had voiced some thoughts about this project after discovering a similar project. Personally, I thought this still had a very unique use-case, specifically in the project management space. 

Before officially clearing this project from my brain, I wanted to check in and see if any interest still exists or if we are officially calling it dead? It did no last long, but I did quite like the idea.

**Comment by virgiliolino** [comment]
Hi Andrew, when conceiving this idea I did a mistake on releasing it too fast to the opensource development process. Think of the DLT's we know pretty well. Their consensus work because there is a good amount of decentralization, but also alignment. Without alignment it would be a mess of forks since day one. 
I experimented lack of alignment and complexity on sharing my vision.
The project is going on, the vision is concretizing. Whenever I perceive the right amount of maturity I will gradually decentralize.
Virgilio

**Reply by Cooksey99** [reply]
You have a good point there. Well, I will keep this project on my radar and you can reach out if you ever want a helping hand. I appreciate your response.
